XenModel/Messages.resx

```diff
@@ -11974,6 +11974,9 @@ Check your settings and try again.</value>
   <data name="UPDATES_WIZARD_LOCAL_STORAGE" xml:space="preserve">
     <value>{0}: The VM '{1}' uses local storage and cannot be migrated.</value>
   </data>
+  <data name="UPDATES_WIZARD_MASTER_VERSION_NOT_COMPATIBLE_PROBLEM" xml:space="preserve">
+    <value>{0}: Master version is not compatible for this update. Please include master in the selection and try again.</value>

```

Comment:
This message is conflicting, it says that the master's version is incompatible with the update but suggests to include the master in the selection.
How about: "The update cannot be applied on slaves without being applied on the master" or "The update needs to also be applied on the master"